docs/admin/serving/feature-flags.md

```diff
@@ -4,91 +4,92 @@ The Knative API is designed to be portable, and abstracts away specific implemen
 
-Alpha means:
+Enabled
+: The feature or extension is enabled and currently in use, or active.
 
-* Might be buggy. Enabling the feature may expose bugs.
-* Support for feature may be dropped at any time without notice.
-* The API may change in incompatible ways in a later software release without notice.
-* Recommended for use only in short-lived testing clusters, due to increased risk of bugs and lack of long-term support.
+Allowed
+: The feature or extension is enabled and can be used or made active, for example, by using an additional annotation or spec configuration for a resource.
 
-Beta means:
+Disabled
+: The feature cannot be used.
 
-* The feature is well tested. Enabling the feature is considered safe.
-* Support for the overall feature will not be dropped, though details may change.
-* The schema and/or semantics of objects may change in incompatible ways in a subsequent beta or stable release. When this happens, we will provide instructions for migrating to the next version. This may require deleting, editing, or re-creating API objects. The editing process may require some thought. This may require downtime for applications that rely on the feature.
-* Recommended for only non-business-critical uses because of potential for incompatible changes in subsequent releases. If you have multiple clusters that can be upgraded independently, you may be able to relax this restriction.
+## Lifecyle
 
-General Availability (GA) means:
+When features and extensions are introduced to Knative, they follow a lifecycle of three stages:
 
-* Stable versions of features/extensions will appear in released software for many subsequent versions.
+Alpha stage
+: Might be buggy. Enabling the feature may expose bugs.
+: Support for feature may be dropped at any time without notice.
+: The API may change in incompatible ways in a later software release without notice.
+: Recommended for use only in short-lived testing clusters, due to increased risk of bugs and lack of long-term support.
 
-# Feature
+Beta stage
 
-Features use flags to safely introduce new changes to the Knative API. Eventually, each feature will graduate to become fully part of the Knative API, and the flag guard will be removed.
+: The feature is well tested. Enabling the feature is considered safe.
+: Support for the overall feature will not be dropped, though details may change.
+: The schema and/or semantics of objects may change in incompatible ways in a subsequent beta or stable release. When this happens, we will provide instructions for migrating to the next version. This may require deleting, editing, or re-creating API objects. The editing process may require some thought. This may require downtime for applications that rely on the feature.
+: Recommended for only non-business-critical uses because of potential for incompatible changes in subsequent releases. If you have multiple clusters that can be upgraded independently, you may be able to relax this restriction.
```

Comment:
```suggestion
: Recommended for only non-business-critical uses because of the potential for incompatible changes in subsequent releases. If you have multiple clusters that can be upgraded independently, you might be able to relax this restriction.
```